src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs

```diff
@@ -467,6 +467,13 @@ public IClassificationLoss CreateComponent(IHostEnvironment env)
         /// <param name="decreaseLearningRate">Decrease learning rate as iterations progress.</param>
         /// <param name="l2Regularization">The L2 weight for <a href='tmpurl_regularization'>regularization</a>.</param>
         /// <param name="numberOfIterations">The number of passes through the training dataset.</param>
+        /// <example>
+        /// <format type="text/markdown">
+        /// <![CDATA[
+        /// [!code-csharp[OnlineGradientDescent](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Regression/OnlineGradientDescent.cs)]
+        /// ]]>
```

Comment:
please don't include this sample until we fix #2425

Reply:
There are two major meanings to have a sample.
1) Learn how to call it
2) Explore from that sample.
A sample with bad prediction ability doesn't break any of them.

---
In reply to: [275564038](https://github.com/dotnet/machinelearning/pull/3319#discussion_r275564038) [](ancestors = 275564038)